test/fixtures/user-app/server/server.js

```diff
@@ -0,0 +1,10 @@
+var loopback = require('loopback');
+var boot = require('loopback-boot');
+var app = module.exports = loopback();
+app.enableAuth();
+boot(app, __dirname);
+app.use(loopback.token({model: app.models.AccessToken}));
+var apiPath = '/';
+app.use(apiPath, loopback.rest());
+app.use(loopback.urlNotFound());
```

Comment:
Including these 2 lines probably isn't going to reflect a regular application using json api component since we disable the default error handling here: https://github.com/digitalsadhu/loopback-component-jsonapi/blob/master/lib/headers.js#L18 and register our own here: https://github.com/digitalsadhu/loopback-component-jsonapi/blob/master/lib/errors.js#L7